raw_data/preprocessed/africa/zanzibar_english.txt

```diff
@@ -0,0 +1,265 @@
+
+(69) [1] No person shall excavate, remove, transport or sell any nonrenewable natural resource from public and/ or private land without a permit as prescribed by the institution responsible for nonrenewable natural resources. [2] No licensing institution shall issue a permit to excavate, remove, transport or sell any nonrenewable natural resource unless: [a] such activity is in accordance with the provisions of this Act and any other law on the matter; [b] the prescribed fee has been set in accordance with the purposes of this Act, especially section 8; [c] the prescribed fee has been paid; [d] the permit allows removal of the nonrenewable natural resource only from a designated excavation site; [e] the permit conditions for large-scale activities require environmental mitigation or restoration of the removal site; and [f] an environmental performance bond has been posted as prescribed. [2] The Minister shall designate the institution responsible for nonrenewable natural resources by order in the Official Gazette. [3] The Minister responsible for nonrenewable natural resources, in collaboration with the institution responsible for the environment, shall designate by notice in the Official Gazette excavation sites for the removal of nonrenewable natural resources. [4] The permit holder shall on demand produce the permit to any authorized officer of the institution responsible for nonrenewable natural resources and any environmental officer. [5] A record of permits shall be kept at the institution designated under subsection [3]. [6] The record in subsection [6] shall be reported on a semi-annual basis to the institution responsible for the environment. 
```

Comment:
Please replace `The record in subsection [6]` with `The record in subsection [5]`.